lib/Ix/DBIC/ResultSet.pm

```diff
@@ -945,10 +945,21 @@ sub ix_get_list ($self, $ctx, $arg = {}) {
     return $error;
   }
 
+  # XXX - make sure isActive is true on all joined tables?
```

Comment:
Yeah, I don't think we currently can trigger this anywhere in topicbox, but perhaps I should solve for it now.